### DISCUSSION TITLE
I am having This problem while I run the app

### DISCUSSION THREAD

**Discussion by shary998** [florisboard/florisboard#1804] [Q&A] [answered]
Error while executing: am start -n "dev.patrickgold.florisboard/dev.patrickgold.florisboard.SettingsLauncherAlias" -a android.intent.action.MAIN -c android.intent.category.LAUNCHER
Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] cmp=dev.patrickgold.florisboard/.SettingsLauncherAlias }
Error type 3
Error: Activity class {dev.patrickgold.florisboard/dev.patrickgold.florisboard.SettingsLauncherAlias} does not exist.

Error while Launching activity
Failed to launch an application on all devices

**Comment by patrickgold** [comment, marked as answer]
If you build the app and install it it is most likely the debug verion, so you need to use "dev.patrickgold.florisboard.debug" as the package name
